nomenclature/processor/data_validator.py

```diff
@@ -17,7 +18,17 @@
 logger = logging.getLogger(__name__)
 
 
-class DataValidationCriteriaValue(IamcDataFilter):
+class WarningEnum(str, Enum):
+    high = "high"
+    medium = "medium"
+    low = "low"
```

Comment:
Suggest to use colors because that is closer to future visualization tools and also more easily to expand in the future (like adding purple, orange, blue) for different interpretations.
```suggestion
    red = "red"
    yellow = "yellow"
```

Reply:
Even though it might be closer to the visualization, I'm not sure that in written form "red" is more understandable than "high" in terms of severity. 
This would mean we'd have validation criteria files like:

```yaml
- variable: Primary Energy
  year: 2010
  upper_bound: 2.5
  lower_bound: 1
  warning_level: red
- variable: Primary Energy
  year: 2010
  upper_bound: 2.5
  lower_bound: 1
  warning_level: yellow
```

which I'm not sure is nicer to read than using _low_, _medium_ and _high_.

What I'd suggest is to keep the levels of _low_, _medium_ and _high_ but actually use numerical values inspired by the way that python logging implements their levels (https://docs.python.org/3/library/logging.html#logging-levels):

```python
class WarningLevel(int, Enum):
    low = 10
    medium = 20
    high = 30
    error = 40
```

in addition, I'd use the error level as the default for `DataValidationCriteria`:
```python
class DataValidationCriteria(IamcDataFilter):
    warning_level: WarningLevel = WarningLevel.error
```

this makes it easier to understand that the default warning level is to raise an error in case validation fails. 
Further down in the code you can then check:

```python
if item.warning_level == WarningLevel.error:
    raise Error
```

in my opinion, this is easier to read and comprehend than using `None` to mean "raise an error".

Reply:
As discussed in person, we will keep the levels as they are less ambiguous, especially if accompanied by a longer description to be implemented later on.